index.html

```diff
@@ -64,136 +65,12 @@ <h1 class="text-titleH1 text-title item-1">Vanessa Bonito</h1>
             <div class="headline-decoration"></div>
         </section>
         <!--Works section-->
-        <section id="works-section" class="grid-works">
+        <section id="dinamicProject" class="grid-works">
```

Comment:
- [ ] Kindly check on your popup desktop version, there are arrows missing, and your technologies list alignment should be similar to your picture and title, also everything should fit without scrolling: reduce the size of your popup. Below is a comparison between your design and Figma design

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/174086225-567f8edd-b27f-401a-be55-6fe8daafa6ce.png) |  ![image](https://user-images.githubusercontent.com/61408860/174086559-8e9c6e55-26c4-479e-a48d-43e0436b4c3c.png)

- [ ] Kindly ensure the icons in the `see live` button and see `source version` are visible. Below is a comparison with the Figma design:

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/174087091-62d58534-8abb-46b2-acdc-c99e2cc57d34.png) |  ![image](https://user-images.githubusercontent.com/61408860/174087129-f852a9ec-c482-4250-a28e-ac6223b1b7f9.png)

Reply:
Hello, @danmainah 
Thanks for your advice, I agree with the tech list alignment and also with the icons in the modals button and I am working on solving it.
But I think the modal in my template actually needs the overflow, please take a loot of this image
![image](https://user-images.githubusercontent.com/53547956/174120104-be4578fe-fecd-4fc5-87e3-ec776bb799f3.png)
What do you think?